lib/Styles/Core.less

```diff
@@ -167,3 +167,15 @@ select {
   font-size: 14px;
   padding-left: 5px;
 }
+
+
+.sr-only{
```

Comment:
Oh ok, I hadn't run into that before.  I guess technically you're not using it here for content intended for screen readers, but more as a general "don't show this thing".  But it's fine in any case.